### PR TITLE
updated story-behind.md

### PR DESCRIPTION
**Pull request by sercanerhan**
# Description

I have updated the docs/story-behind.md file to reflect my suggestion. This update is not mandatory and can be ignored if it doesn't align with the goals of the project.

**discord username: sercanerhan#0543**

**closes #issue**

Please describe your changes. Also describe your aim and content. Do not forget to list the dependencies required caused by those changes.

## Things to check before creating a PR
- [ ] I have inspected my topic, checked.
- [ ] If it is a core feature, executed detailed tests.

## Creating PR rules
- [ ] PR must be created for an issue with approved tag. Otherwise PR will be rejected.
- [ ] Relevant issue number: The issue number related to PR must be attached to head of PR header, after prefix after prefix # must be attached in parenthesis. A header like this could be used "prefix(#issue_number): PR header" .
- [ ] A descriptive and understandable title: The PR title should clearly describe the nature and purpose of the changes. The PR title should be the first thing displayed when the PR is opened. And it should follow the semantic commit rules. For example, a title like "docs(#issueId): Add README.md" can be used.
- [ ] Related file selection: Only relevant files should be touched and no other files should be affected.
- [ ] Format and Lint Suitability : The code should be made in accordance with a certain format standard and examined according to the lint rules.
- [ ] Clean commit history: The commits where the changes are made should be clear and organized.
- [ ] Opening PR when job is completed: PR should be opened when job is completed and sent for review by other team members.

## Changes

- [ ] A new feature (a change that adds a new feature, not a breaking change)
- [ ] A new refactor (a change that is not a breaking change, that improves the readability or performance of the code)
- [ ] A breaking change
- [x] Documentation change

# How were these changes tested?

Please describe the tests you did to test the changes you made. Please also specify your test configuration.

**Test Configuration**:

* Firmware version:
* Hardware:
* Toolchain:
* SDK:

### CODE REVIEW

**Comment by sercanerhan** [comment]
@eraygundogmus I noticed your pull request and I have created a pull request of my own to update the documentation. Please feel free to ignore it and delete the pull request if you need.

File: docs/STORY-BEHIND.md [hunk 1]
```diff
@@ -21,11 +29,15 @@ Working together with people they have never met in their field, they have devel
 ![afetharita](./images/afetharita3.png)
 ![afetharita](./images/afetharita4.png)
 
-**In just 2 days.**
-It is impossible to express how much stress, busyness they have experienced.
-Each of them is a hero.
```
Comment (eraygundogmus) [inline docs/STORY-BEHIND.md:26]: This was not a paragraph itself, but I am still open to merge this PR if you could change "## In just 2 days"

Thanks for you effort!

**Review by eraygundogmus — CHANGES_REQUESTED** [review]

File: docs/STORY-BEHIND.md (hunk 1 above)
Comment (sercanerhan) [reply docs/STORY-BEHIND.md:26]: @eraygundogmus within a new commit for the same branch, I've removed the title and update the text with new suggestion.

**Comment by usirin** [comment]
@sercanerhan closing this for now, let's reopen this after emergency mode.

**Pull request closed without merging**